app/pulp/app/management/commands/reset-admin-password.py

```diff
@@ -0,0 +1,39 @@
+from gettext import gettext as _
+from getpass import getpass
+
+from django.core.management import BaseCommand, CommandError
+
+from pulp.app.models import User
+
+
+class Command(BaseCommand):
+    """
+    Django management command for resetting the password of the 'admin' user in Pulp.
+    """
+    help= _('Resets "admin" user\'s password.')
+
+    def add_arguments(self, parser):
+        parser.add_argument('--random',
+            action='store_true',
+            dest='random',
+            default=False,
+            help='Generate random password for \'admin\' user.')
+
+    def handle(self, *args, **options):
+        user = User.objects.get_or_create(username='admin')[0]
+        if options['random']:
+            password = User.objects.make_random_password(length=20)
+            user.set_password(password)
+            user.save()
+            self.stdout.write('Successfully set "admin" user\'s password to "%s"' % password)
+        else:
+            password = getpass('Please enter new password: ')
+            password2 = getpass('Please enter new password again: ')
+            if not password:
+                raise CommandError("The password must be at least 1 character long.")
+            if password == password2:
+                user.set_password(password)
+                user.save()
+                self.stdout.write('Successfully set password for "admin" user.')
+            else:
+                raise CommandError('The passwords did not match.')
```

Comment:
Django docs recommend using self.stdout.write to handle all output from the command.

https://docs.djangoproject.com/en/1.8/howto/custom-management-commands/#module-django.core.management

look at the first note